src/SFA.DAS.EmployerFinance/Commands/RefreshEmployerLevyData/RefreshEmployerLevyDataCommandHandler.cs

```diff
@@ -55,22 +54,16 @@ protected override async Task HandleCore(RefreshEmployerLevyDataCommand message)
 
             foreach (var employerLevyData in message.EmployerLevyData)
             {
-                var declarations = employerLevyData.Declarations.Declarations.OrderBy(c => c.SubmissionDate).ToArray();
+                var declarations = await _levyImportCleanerStrategy.Cleanup(employerLevyData.EmpRef, employerLevyData.Declarations.Declarations);
 
-                declarations = FilterDuplicateHmrcDeclarations(employerLevyData.EmpRef, declarations);
+                var temp = declarations.ToArray();
```

Comment:
Rename temp or change the return type of Cleanup.

Reply:
I've changed the return type.